mainframer.sh

```diff
@@ -89,7 +96,7 @@ function syncBeforeRemoteCommand {
 	eval "$COMMAND"
 
 	endTime="$(date +%s)"
-	echo "Sync done: took $((endTime-startTime)) seconds."
+	echo "Sync done: took $(convertsecs $((endTime-startTime)))"
```

Comment:
stupid question, but shouldn't this be `convertSecondsToTime` instead of `convertsecs`?